app/controllers/application_controller.rb

```diff
@@ -48,7 +48,15 @@ def peek_enabled?
   private
 
   def current_user
-    @current_user ||= User.find_or_create_by(email_address: session[:user]['email']) if session[:user] && session[:user]['email']
+    @current_user ||= find_current_user
+  end
+
+  def find_current_user
+    if ENV['FORCE_USER_ID']
```

Comment:
> In order simulate what a user sees

Have you needed to do this regularly? Seems like the real issue here is a lack of admin tools on production.

>  in dev when using a db snapshot from production

Not sure that we should be making this typical practice...not sure how sensitive our clients are about their data. They might not be thrilled about copies of the database roaming around on our laptops.